### DISCUSSION TITLE
Just curious to know how you set ideal value of tracking formula constants?

### DISCUSSION THREAD

**Discussion by vasubits** [rsms/inter#573] [Q&A] [answered]
Hi, Thanks for designing Inter Font. I have one question about Inter font. How did you figure out ideal values for a,b and c  [https://rsms.me/inter/dynmetrics/](https://t.co/O0W1tH045N)

**Comment by rsms** [comment, marked as answer]
1. I came up with good tracking values for a bunch of optical sizes (the "ideal values" in the sidebar)
2. Made a simple interactive graph tool (the graph above "ideal values" in the sidebar)
3. Trial and error until I found a fitting curve

<img width="253" alt="Screen Shot 2024-03-26 at 13 51 34" src="https://github.com/rsms/inter/assets/47046/25d7d588-ba34-4c48-a4ae-9a065158e4c9">

https://d.rsms.me/inter-website/v3/dynmetrics/
